rules.d/60-persistent-storage.rules

```diff
@@ -124,6 +124,6 @@ ENV{ID_PART_ENTRY_SCHEME}=="gpt", ENV{ID_PART_ENTRY_NAME}=="?*", SYMLINK+="disk/
 # by-diskseq link (if an app is told to open a path like this, they may parse
 # the diskseq number from the path, then issue BLKGETDISKSEQ to verify they really got
 # the right device, to access specific disks in a race-free fashion)
-ENV{DISKSEQ}=="?*", ENV{DEVTYPE}!="partition", ENV{DISKSEQ}=="?*", SYMLINK+="disk/by-diskseq/$env{DISKSEQ}"
+ENV{DISKSEQ}=="?*", ENV{DEVTYPE}!="partition", ENV{DISKSEQ}=="?*", ENV{ID_IGNORE_DISKSEQ}!="?*", SYMLINK+="disk/by-diskseq/$env{DISKSEQ}"
```

Comment:
shouldn't the test use `!="1"`?

i.e. it's a 0/1 flag, and that's how we test fot other siilar props, i.e. ID_BTRFS_READY, ID_PART_GPT_AUTO_ROOT, ID_INPUT_ACCELEROMETER and so on...

Reply:
Then, sd_device_open() should also check the value??